examples/data-objects/webflow/src/index.ts

```diff
@@ -6,10 +6,10 @@
 import { ContainerViewRuntimeFactory } from "@fluid-example/example-utils";
 import React from "react";
 
-export { FlowDocument } from "./document";
-export { Editor } from "./editor";
-import { WebFlow, WebflowView } from "./host";
-export { htmlFormatter } from "./html/formatters";
+export { FlowDocument } from "./document/index.js";
+export { Editor } from "./editor/index.js";
+import { WebFlow, WebflowView } from "./host/index.js";
+export { htmlFormatter } from "./html/formatters.js";
```

Comment:
Not really. My understanding of the more recent history of it is that:

`index.js` files are handled specially by the CJS module loader in that an import to `./foo` would attempt to look for files `./foo.js`, `./foo/index.js`, and analogous files with any extensions registered by `require.extensions`.
I'm guessing this behavior came out of the fact that JS package publishers did not have good ways to "map" the files in their project to valid imports for consumers, so an index convention was settled.

Since then, the esnext format has largely solved that problem (packages can expose subpaths, import specifications are much more well-behaved and understood across the ecosystem) and so does away with the more egregious artifacts of cjs loaders.

All these changes do is basically make "where is this file located" more explicit. You'll still get the same compile-time validation if you try to import from `./has-no-index/index.js` but that folder is bogus.